Makefile

```diff
@@ -169,7 +169,10 @@ codegen: ## Auto generate files based on AWS APIs response
 stable-release-pr: ## Generate PR for stable release
 	$(WITH_GOFLAGS) ./hack/release/stable-pr.sh
 
-release: ## Builds and publishes stable release if env var RELEASE_VERSION is set, or a snapshot release otherwise
+snapshot:
+	$(WITH_GOFLAGS) ./hack/release/snapshot.sh
+
+release: ## Builds and publishes stable release if env var RELEASE_VERSION is set
```

Comment:
Is this statement about the RELEASE_VERSION still true or does it matter whether the current commit is tagged?

Reply:
It's not actually been true, `release.sh` has been using the git tag to determine if it's a stable release rather than using `RELEASE_VERSION`. The current version will always create a release, though it might break if there isn't a valid git tag. Might be good to add a check for that.